app/services/workflow_find_service.rb

```diff
@@ -7,13 +7,13 @@ def find(tag_attrs)
     Workflow.where(:id => workflow_ids)
   end
 
-  # find workflows from a collection of [app_name, object_type, [namespace, key, value]]
+  # find workflows from a collection of [app_name, object_type, [tags]]
   def find_by_tag_resources(tag_resources)
     return [] if tag_resources.blank?
 
     query = nil
     tag_resources.each_with_index do |tr, i|
-      tag_names = tr['tags'].map { |tag| "/#{tag['namespace']}/#{tag['name']}=#{tag['value']}" }
+      tag_names = tr['tags'].collect { |tag| tag[:tag] }
```

Comment:
not sure why we have `tag[:tag]`. shouldn't be `tag['tag']`?
Don't understand why we need to have format of `[{"tag": "tag1"}, {"tag": "tag2"}]`, which can be simply `["tag1", "tag2"]`. I realize this format comes from external services, but still don't understand why.